Dockerfile

```diff
@@ -41,14 +41,28 @@ RUN apt-get update && \
   ffmpeg \
   libavdevice-dev && \
   rm -rf /var/lib/apt/lists/*
+# install required libs for building graphviz
+RUN apt-get update && \
+  apt-get install -y libtool \
+  libltdl-dev \
+  automake \
+  autoconf \
+  bison \
+  flex \
+  tcl \
+  libperl-dev \
+  python-dev-is-python3 \
+  libpython3-dev \
+  dh-python \
+  ghostscript \
+  libgd-dev \
+  fontconfig \
+  libcairo2-dev \
+  libpango1.0-dev \
+  libgts-dev && \
+  rm -rf /var/lib/apt/lists/*
 
 WORKDIR /workspace/
-# install megatron core, this can be removed once 0.3 pip package is released
-RUN git clone https://github.com/NVIDIA/Megatron-LM.git && \
-  cd Megatron-LM && \
-  git checkout ab0336a5c8eab77aa74ae604ba1e73decbf6d560 && \
-  pip install -e .
-
 # Install megatron core, this can be removed once 0.3 pip package is released
```

Comment:
Commit id doesn't match above

Reply:
There are 2 installation of Megatron-LM in main: with `ab0336a5c8eab77aa74ae604ba1e73decbf6d560` and `375395c187ff64b8d56a1cd40572bc779864b1bd`.
Which one is correct?
https://github.com/NVIDIA/NeMo/blob/main/Dockerfile#L47
https://github.com/NVIDIA/NeMo/blob/main/Dockerfile#L54